src/App/Types.hs

```diff
@@ -83,6 +89,54 @@ data ProjectRevision = ProjectRevision
 instance ToJSON ProjectRevision where
   toEncoding = genericToEncoding defaultOptions
 
+data ReleaseGroupRevision = ReleaseGroupRevision
+  { releaseGroupTitle :: Text
+  , releaseGroupReleaseRevision :: ReleaseGroupReleaseRevision
+  , releaseGroupLicensePolicy :: Maybe Text
+  , releaseGroupSecurityPolicy :: Maybe Text
+  , releaseGroupQualityPolicy :: Maybe Text
+  , releaseGroupTeams :: Maybe [Text]
+  }
+  deriving (Eq, Ord, Show, Generic)
+
+instance ToJSON ReleaseGroupRevision where
+  toJSON ReleaseGroupRevision{..} =
+    object
+      [ "title" .= releaseGroupTitle
+      , "release" .= releaseGroupReleaseRevision
+      , "licensePolicy" .= releaseGroupLicensePolicy
+      , "securityPolicy" .= releaseGroupSecurityPolicy
+      , "teams" .= releaseGroupTeams
+      ]
```

Comment:
[Optional] It's annoying that you have to write this. GHC has the [disambiguate record fields](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/disambiguate_record_fields.html) extension which I *think* would allow you to just name the fields the same thing as their JSON counterparts and then you can generate the To/From JSON instances with Generics.

I understand that it's a departure from how we've done things, but I think it's safe to do. `DisambiguateRecordFields` will be on by default in the `GHC2024` language edition. The one caveat is that to define two record types with the same field names, the records have to go in different modules. This might make it not worth doing the work for now.